pkg/api/converterfromplugin_test.go

```diff
@@ -46,10 +46,11 @@ func internalPluginConfig() Config {
 				Key:  tls.GetDummyPrivateKey(),
 			},
 		},
+		RootDockerConfig: []byte("RootDockerConfig"),
 		// Container images configuration
 		Images: ImageConfig{
-			GenevaImagePullSecret:        []byte("Images.GenevaImagePullSecret"),
```

Comment:
Is your linting working fine? These changes do not look right to me.

Reply:
golint was being difficult, fixed now